### PR TITLE
chore(config): Add environment config options

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by jbuck** [comment]
If anyone has any idea why this is failing with the following error:

```
$ CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt build

>> Local Npm module "grunt-amdcheck" not found. Is it installed?
>> Local Npm module "grunt-bump" not found. Is it installed?
>> Local Npm module "grunt-contrib-csslint" not found. Is it installed?
>> Local Npm module "grunt-contrib-watch" not found. Is it installed?
>> Local Npm module "grunt-conventional-changelog" not found. Is it installed?
>> Local Npm module "grunt-copyright" not found. Is it installed?
>> Local Npm module "grunt-eslint" not found. Is it installed?
>> Local Npm module "grunt-htmllint" not found. Is it installed?
>> Local Npm module "grunt-jsonlint" not found. Is it installed?
>> Local Npm module "grunt-newer" not found. Is it installed?
>> Local Npm module "grunt-nsp" not found. Is it installed?
>> Local Npm module "grunt-sass-lint" not found. Is it installed?
>> Local Npm module "grunt-todo" not found. Is it installed?
Loading "Gruntfile.js" tasks...ERROR
>> SyntaxError: Unexpected token }
Warning: Task "build" not found. Use --force to continue.

Aborted due to warnings.


Execution Time (2017-03-27 23:50:26 UTC-0)
loading tasks  1.6s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 100%
Total 1.6s


CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt build returned exit code 3

Action failed: CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt build
```

File: server/lib/configuration.js [hunk 1]
```diff
@@ -420,22 +428,26 @@ const conf = module.exports = convict({
...
-      format: String
+      format: String,
+      env: 'SENTRY_ENDPOINT'
     },
     server_errors_dsn: {
       default: undefined,
       doc: 'Sentry config for Express server-side errors. If not set, then no errors reported.',
-      format: String
+      format: String,
+      'SENTRY_SERVER_ERRORS_DSN'
```
Comment (seanmonstar) [inline server/lib/configuration.js]: Missing property name here

**Comment by jbuck** [comment]
oh my goodness. Thank you!

Time to close the ol laptop for the day

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a6923f386b310febd50671a2b0f2bf389bbf82ec
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: server/lib/configuration.js [hunk 2]
```diff
@@ -19,7 +19,8 @@ const conf = module.exports = convict({
   allowed_parent_origins: {
     default: [],
     doc: 'Origins that are allowed to embed FxA within an IFRAME',
-    format: Array
+    format: Array,
+    env: 'ALLOWED_PARENT_ORIGINS'
```
Comment (seanmonstar) [inline server/lib/configuration.js]: Seems the linting rules dislike that the property names are not in alphabetical order, such as `env` should be before `format`. All the way down.

**Comment by jbuck** [comment]
@seanmonstar all green now!

**Pull request merged**